frontend/public/components/monitoring.tsx

```diff
@@ -1224,16 +1200,18 @@ const RulesPage_ = (props) => (
     Header={ruleTableHeader}
     kindPlural="Alerting Rules"
     nameFilterID="alerting-rule-name"
+    labelFilter="alerts"
     reduxID="monitoringRules"
     Row={RuleTableRow}
     rowFilter={rulesRowFilter}
+    labelPath="labels"
```

Comment:
@kyoto we can pass the path for silences similarly and activate label filtering on it.